spec/user_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+

```

Comment:
- [ ] All methods in your User model should be tested as mentioned in the project requirements. Please write a test for the method `latest_posts`.